src/components/MoreInformation.js

```diff
@@ -53,12 +55,100 @@ function ExtraData({ data }) {
     } else if (typeof data == "object") {
         let elements = [];
         for (const key in data) {
-            elements.push(
-                <div className={classes.extraData}>
-                    <b>{key}:</b> {data[key]}
-                </div>
-            );
+            //formats additional information instead of just adding it to the more information section
+            if (key === "Additional Information") {
+                const currentData = data[key];
+                if (currentData) {
+                    var finalData = parseMD(currentData);
+                    //adds the parsed data as straight html
+                    elements.push(
+                        <div
+                            className={classes.extraData}
+                            dangerouslySetInnerHTML={{
+                                __html: `<b>${key}:</b> ${finalData}`,
+                            }}
+                        />
+                    );
+                }
+            } else {
+                elements.push(
+                    <div className={classes.extraData}>
+                        <b>{key}:</b> {data[key]}
+                    </div>
+                );
+            }
         }
         return elements;
     }
 }
+
+function parseMD(currentData) {
+    //encodes angle brackets to escape data
+    let workingData1;
+    let workingData2;
+    let indexOfLAB = currentData.indexOf("<");
+    let indexOfRAB = currentData.indexOf(">");
+    if (indexOfLAB !== -1) {
+        workingData1 = currentData.replaceAll("<", "&#60;");
+    } else {
+        workingData1 = currentData;
+    }
+    if (indexOfRAB !== -1) {
+        workingData2 = workingData1.replaceAll(">", "&#62;");
+    } else {
+        workingData2 = workingData1;
+    }
```

Comment:
seems we can call `replaceAll` on a string without any occurrences of the replacement, so you can just always call that function regardless of if there's a > or <.

I agree with @MattRC7; if we can just avoid using `dangerouslySetInnerHTML` then that solves our problem. I'm cool with using a third-party library if there's one that looks like a good fit; makes our code less bulky.